client/my-sites/stats/activity-log-switch/index.jsx

```diff
@@ -0,0 +1,125 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { includes } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+import ActivityLogRewindToggle from 'my-sites/stats/activity-log/activity-log-rewind-toggle';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+import { getRewindState } from 'state/selectors';
+
+class ActivityLogSwitch extends Component {
+	static propTypes = {
+		siteId: PropTypes.number.isRequired,
+
+		// Connected props
+		siteSlug: PropTypes.string.isRequired,
+		rewindState: PropTypes.string.isRequired,
+		failureReason: PropTypes.string.isRequired,
+
+		// localize
+		translate: PropTypes.func.isRequired,
+	};
+
+	state = {
+		dismissed: false,
+	};
+
+	/**
+	 * Dismiss the dialog forever.
+	 *
+	 * @returns {undefined}
+	 */
+	// ToDo: replace with real method to dismiss and store in server
+	noThanksDismissIt = () => this.setState( { dismissed: true } );
+
+	/**
+	 * Renders the main button whose functionality and label varies depending on why Rewind is unavailable.
+	 *
+	 * @returns {object} Primary button.
+	 */
+	getMainButton() {
+		const {
+			siteId,
+			siteSlug,
+			translate,
+		} = this.props;
+		switch ( this.props.failureReason ) {
+			case 'vp_can_transfer':
+				return (
+					<ActivityLogRewindToggle
+						siteId={ siteId }
+						label={ translate( 'Switch now' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **8** times:
`translate( 'Switch' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).